examples/backupRTC/backupRTC.ino

```diff
@@ -0,0 +1,105 @@
+/*
+  backupRTC
```

Comment:
Rename it to explicitly told that it is for F1. Ex: `F1DateRetention`

Reply:
changed